docs/content/en/docs/design/config.md

```diff
@@ -10,15 +10,74 @@ You can configure Skaffold with the Skaffold configuration file,
 project directory; when you run the `skaffold` command, Skaffold will try to
 read the configuration file from the current directory.
 
-`skaffold.yaml` consists of five different components:
+`skaffold.yaml` consists of several different components:
 
 | Component  | Description |
 | ---------- | ------------|
 | `apiVersion` | The Skaffold API version you would like to use. The current API version is {{< skaffold-version >}}. |
 | `kind`  |  The Skaffold configuration file has the kind `Config`.  |
+| `metadata`  |  Holds additional properties like the `name` of this configuration.  |
 | `build`  |  Specifies how Skaffold builds artifacts. You have control over what tool Skaffold can use, how Skaffold tags artifacts and how Skaffold pushes artifacts. Skaffold supports using local Docker daemon, Google Cloud Build, Kaniko, or Bazel to build artifacts. See [Builders](/docs/pipeline-stages/builders) and [Taggers]({{< relref "/docs/pipeline-stages/taggers" >}}) for more information. |
 | `test` |  Specifies how Skaffold tests artifacts. Skaffold supports [container-structure-tests](https://github.com/GoogleContainerTools/container-structure-test) to test built artifacts. See [Testers]({{< relref "/docs/pipeline-stages/testers" >}}) for more information. |
 | `deploy` |  Specifies how Skaffold deploys artifacts. Skaffold supports using `kubectl`, `helm`, or `kustomize` to deploy artifacts. See [Deployers]({{< relref "/docs/pipeline-stages/deployers" >}}) for more information. |
 | `profiles`|  Profile is a set of settings that, when activated, overrides the current configuration. You can use Profile to override the `build`, `test` and `deploy` sections. |
+| `requires`|  Specifies a list of other skaffold configurations to import into the current config |
 
 You can [learn more]({{< relref "/docs/references/yaml" >}}) about the syntax of `skaffold.yaml`.
+
+## Configuration dependencies
+
+In addition to authoring pipelines in a skaffold configuration file, we can also import pipelines from other existing configurations as dependencies. Skaffold manages all imported and defined pipelines in the same session. It also ensures all artifacts in a required configs are built prior to those in current config (provided the artifacts have dependencies defined); and all deploys in required configs are applied prior to those in current config.
+
+### Local config dependency
+
+This is how a dependency added to configurations `cfg1` and `cfg2` that are defined in another `skaffold.yaml` file, looks like:
```

Comment:
Could you add that `cfg1` and `cg2` are defined in `metadata.name` of the other `skaffold.yaml`? It took me a bit to remember that's how they are defined.

Reply:
good point. done.